spec/controllers/alerts_list_controller_spec.rb

```diff
@@ -7,14 +7,27 @@
 
   describe '#index' do
     let!(:alert) { create(:alert) }
+    let!(:another_alert) { create(:alert, resolved: true) }
+
     context 'for admins' do
       render_views
       before { allow(controller).to receive(:current_user).and_return(admin) }
+
       it 'renders the alerts list' do
         get :index
+
         expect(response.status).to eq(200)
         expect(response.body).to have_content(alert.type)
       end
+
+      it 'should not render resolved alerts' do
```

Comment:
Rather than hiding resolved alerts altogether, it would be better to still show them, but indicate that they are resolved.